code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -312,6 +326,9 @@
 
 /obj/item/weapon/reagent_containers/food/drinks/soda_cans
 	name = "soda can"
+	flags = FALSE
+	spillable = 0
+	var/list/open_sounds = list('sound/effects/can_open1.ogg', 'sound/effects/can_open2.ogg', 'sound/effects/can_open3.ogg')
```

Comment:
the reason it was like that in /vg/ code, there was a comment saying "so admins can change the sounds"